installer/main.sh

```diff
@@ -366,6 +368,15 @@ addtrap "stty echo 2>/dev/null"
 BIN="$PREFIX/bin"
 CHROOTS="$PREFIX/chroots"
 
+# Check if space requirements have been met
+# if not, check for dualboot
+if ! check_space "$SIZEMIN" "$AVAILMIN"; then
+    sleep 3
```

Comment:
probably don't want output, delay, more output, more delay.

the other instances we do this, we have extra text that says something like "installation will continue in 5 seconds, or hit ctrl-c to cancel"...copy that text?